app/views/layouts/_param.html.erb

```diff
@@ -0,0 +1 @@
+<%= debug(params.permit!) if Rails.env.development? %>
```

Comment:
cái trang của e nó in ra chỗ debug param là permitted: false nên e thêm .permit xem nó in ra true không nhưng e cũng không biết nó có tác dụng gì không :v